docs/multicluster/user-guide.md

```diff
@@ -2,99 +2,114 @@
+     latest Antrea version, so we won't repeat it in the following steps):
 
      ```bash
      kubectl apply -f multicluster/build/yamls/antrea-multicluster-leader-global.yml
      ```
 
 2. Install Multi-cluster Controller in the leader cluster. Since Multi-cluster Controller is
-running as namespaced deployment, you should create a Namespace first, then apply the
-manifest with new Namespace.
+   running as namespaced Deployment, you should create a Namespace first, then apply the
+   manifest with new Namespace.
 
-* For any given release `<TAG>` (e.g. `v1.5.0`).
-  
   ```bash
-  kubectl create ns antrea-mcs-ns
+  kubectl create ns antrea-multicluster
   curl -L https://github.com/antrea-io/antrea/releases/download/v1.5.0/antrea-multicluster-leader-namespaced.yml >   antrea-multicluster-leader-namespaced.yml
-  sed 's/changeme/antrea-mcs-ns/g' antrea-multicluster-leader-namespaced.yml | kubectl apply -f -
-  ```
-  
-* To deploy the latest version (checkout and built from the `main` branch).
-  
-  ```bash
-  kubectl create ns antrea-mcs-ns
-  multicluster/hack/generate-manifest.sh -l antrea-mcs-ns | kubectl apply -f -
+  sed 's/changeme/antrea-multicluster/g' antrea-multicluster-leader-namespaced.yml | kubectl apply -f -
   ```
-  
-#### Deployment in a Member Cluster
 
-You can run the following commands to install Multi-cluster Controller to all
-member clusters. The command will run the controller in the "member" mode in the `kube-system`
-Namespace by default.
+#### Deploy in a Member Cluster
 
-* For any given release `<TAG>` (e.g. `v1.5.0`).
+You can run the following command to install Multi-cluster Controller in a
+member cluster. The command will run the controller in the "member" mode in the
+`kube-system` Namespace. If you want to use a different Namespace other than
+`kube-system`, you can edit `antrea-multicluster-member.yml` and change
+`kube-system` to the desired Namespace.
 
 ```bash
 kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-multicluster-member.yml
 ```
 
-* To deploy the latest version (checkout and built from the `main` branch).
-
-```bash
-kubectl apply -f multicluster/build/yamls/antrea-multicluster-member.yml
-```
-
-#### Deploy Leader and Member in one Cluster
+#### Deploy Leader and Member in One Cluster
 
-There is no deployment dependency between member and leader clusters if you are using a cluster
-as a dedicated leader cluster. But if you'd like to run both leader and member controllers in
-one cluster, below is the required deployment sequence:
+We need to run two instances of Multi-cluster Controller in the dual-role
+cluster, one in leader mode and another in member mode. There is no deployment
+dependency between member and leader clusters when you have a dedicated leader
+cluster. But to run both leader and member controllers in one cluster, you need
+to follow the deployment sequence below:
```

Comment:
Is it still true? @luolanzone

Reply:
I think there is no dependency even the cluster run both leader and member if user is using 'kubectl apply', but if user is using `kubectl create -f`, CRD installation will fail since some of them are already existing like `ClusterClaim`.

Reply:
If it is true, we should remove the words around ordering.